01-Lab/SoftwareIntro.ipynb

```diff
@@ -35,29 +31,11 @@
    "outputs": [],
    "source": [
     "%%bash\n",
-    "#^the %% symbols tell the notebook to execute bash commands\n",
-    "cd ~/Downloads  #change directories to the Downloads folder, the ~ is shorthand for \"home directory\"\n",
-    "ls  #list the files there"
-   ]
-  },
-  {
-   "cell_type": "raw",
-   "metadata": {},
-   "source": [
-    "%%bash\n",
-    "pwd  #tell me where I am again?\n",
-    "#when I move cells, I moved back to the 01-Lab directory,but I can still move the file to here\n",
-    "mv /home/brain/Downloads/lab01_images.tar.gz ./lab01_images.tar.gz"
-   ]
-  },
-  {
-   "cell_type": "raw",
-   "metadata": {},
-   "source": [
-    "%%bash\n",
-    "ls #is it here?\n",
-    "#unpack a tar file: tar -zxvf tar-archive-name.tar.gz\n",
-    "tar -zxvf lab01_images.tar.gz"
+    "\n",
+    "#the following bash commands will allow us to download the data using command line tools\n",
```

Comment:
add space in between # and word, (i.e. `# the`, not `#the`) as per [pep8 standards](https://www.python.org/dev/peps/pep-0008/#inline-comments).